Components/Flash/Driver/SPIFlash.hpp

```diff
@@ -8,21 +8,21 @@
 #define SPIFLASH_WRAPPER_HPP_
 
 /* Includes ------------------------------------------------------------------*/
+#include "Flash.hpp"
+#include "SystemDefines.hpp"
 #include "w25qxx.hpp"
 #include "w25qxxConf.hpp"
-#include "SystemDefines.hpp"
-#include "Flash.hpp"
 
 /* Constants and Macros -------------------------------------------------------*/
-constexpr uint16_t DEFAULT_FLASH_SECTOR_SIZE = 4096; // 4KB - Default If Flash was not Initialized (to prevent % by 0 errors)
+constexpr uint16_t DEFAULT_FLASH_SECTOR_SIZE =
+    4096;  // 4KB - Default If Flash was not Initialized (to prevent % by 0 errors)
```

Comment:
nit: idk this seems ok but a little weird, I like to have the comment above instead... kinda just preference though so not a huge issue.

Reply:
https://github.com/StudentOrganisationForAerospaceResearch/AvionicsSoftware/pull/86#discussion_r1443815339